integration/base/src/main/java/com/inrupt/client/integration/base/AccessGrantScenarios.java

```diff
@@ -86,20 +83,27 @@ public class AccessGrantScenarios {
     private static String podUrl;
     private static String issuer;
     protected static String webidUrl;
-    private static final String MOCK_USERNAME = "someuser";
-
-    private static final String CLIENT_ID = config.getValue("inrupt.test.client-id", String.class);
-    private static final String CLIENT_SECRET = config.getValue("inrupt.test.client-secret", String.class);
+    protected static String requesterWebidUrl;
+    private static final String MOCK_RESOURCE_OWNER_USERNAME = "someuser";
+    private static final String MOCK_REQUESTER_USERNAME = "requester";
+
+    private static final String REQUESTER_CLIENT_ID = config.getValue("inrupt.test.requester.client-id", String.class);
+    private static final String REQUESTER_CLIENT_SECRET =
+        config.getValue("inrupt.test.requester.client-secret", String.class);
+
+    private static final String RESOURCE_OWNER_CLIENT_ID =
+        config.getValue("inrupt.test.client-id", String.class);
```

Comment:
I would find it less error-prone to align the naming of the environment variable with the change in this PR, and make it `inrupt.test.resource-owner.client-id` (respectively `inrupt.test.resource-owner.client-secret`)

Reply:
I see. I took a peak at the js ones and they are OWNER and REQUESTER. 
I want to do this in a separate PR because I also want to change the OWNER info used in GitHub.